app/views/activities/_detailsModal.html.slim

```diff
@@ -0,0 +1,14 @@
+<div id="detailsModal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
Good enough, but:

* please convert to slim
* try to avoid using predefined names for classes like MyModalLabel and such - make them more domain specific